### PR TITLE
Fix broken documentation build.

### PR DESCRIPTION
**Pull request by awholegunch**
Fix broken doc build:
- Replaces the description of the "none" value
- Updates MD5 hash

Running as [Quick Build 2](http://builds.cask.co/browse/CDAP-DQB78-2)

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -1769,7 +1769,7 @@
 
   <property>
     <name>security.store.provider</name>
-    <value>none</value>
+    <value></value>
```
Comment (wolf31o2) [inline cdap-common/src/main/resources/cdap-default.xml]: No, it's supposed to be `none` as the value, not empty.

**Comment by wolf31o2** [comment]
So what was broken in the build that this fixed? The MD5?

**Comment by awholegunch** [comment]
And the description.

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**